resources/dicts/patrols/new_cat.json

```diff
@@ -466,10 +478,13 @@
         ],
         "antag_fail_outcomes": [
             {
-                "text": "p_l tries to drive the cat away, but they trip and stumble over a loose twig. They lay on the ground and don't get back up. p_l takes a bit of time to bury them - just to keep the scavengers away.",
+                "text": "p_l tries to drive the cat away, but {PRONOUN/n_c:0/subject} {VERB/n_c:0/trip/trips} and {VERB/n_c:0/stumble/stumbles} over a loose twig. {PRONOUN/n_c:0/subject} {VERB/n_c:0/lie/lies} on the ground and {VERB/n_c:0/doesn't/don't} get back up. p_l takes a bit of time to bury {PRONOUN/n_c:0/object} - just to keep the scavengers away.",
```

Comment:
I'm fairly sure its past tense present tense for the verbs!

don't/doesn't in this case!

I probably should have made a review, but I didn't think I'd find much.